schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+ id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
```

Comment:
- [ ] <b>OPTIONAL:</b>   It would be nice to add `PRIMARY KEY` constraint on ID just to make sure that it is a primary key.